QuickDeployment/QuickDeploymentFirewallChanges.md

```diff
@@ -3,34 +3,40 @@
 
 - [External Ports](#external-ports)
 - [Internal Ports](#internal-ports)
-  - [FAQs](#faqs)
-    - [Can I open up the CCM Service's port to allow machines to report in from anywhere?](#can-i-open-up-the-ccm-services-port-to-allow-machines-to-report-in-from-anywhere)
+    - [FAQs](#faqs)
+        - [Can I open up the CCM Service's port to allow machines to report in from anywhere?](#can-i-open-up-the-ccm-services-port-to-allow-machines-to-report-in-from-anywhere)
 
 <!-- /TOC -->
 
 ![Image of CCM Architecture](IMAGE_URL)
 
 ## External Ports
 
-> :warning: **WARNING**: Performing this incorrectly could cause security issues and possibly cause you to be subjected to copyright law/redistribution. Read all of this first. Danger, Will Robinson!
+> :warning: **WARNING**: Performing this incorrectly could cause security issues and possibly cause you to be subjected to copyright law/redistribution.
+> Read all of this first.
+> Danger, Will Robinson!
```

Comment:
No robot? 🤖 

😆

Reply:
I wasn't sure if we wanted that or not. 😝